docs/appendix/yaml.rst

```diff
@@ -360,7 +393,25 @@ observation label is first and the model label is second
 (e.g., ['airnow_cmaq_expt', 'airnow_rrfs_13km', 'airnow_wrfchem_v4.2'])
 
 **data_proc:** This section stores all of the data processing information.
-
+   
+   * **filter_dict:** This is a dictionary used to filter the paired data sent 
+     to the stats routine. The keys of the dictionary should be columns of 
+     of the paired dataset which will be used in filtering. If there are 
+     multiple keys, this will loop over all of them. The value of the dict  
+     should be another dict with keys 'value' and 'oper'. 'value' can be 
+     a single value or list of values used when filtering the data. 
+     'oper' is the operation used when comparing the dataset values.  
+     Examples of operations are ==, !=, >, >=, etc. Additionally, when 
+     comparing to a list, "oper" can be set to "isin" or "isnotin" to filter 
+     by values in the list or not in the list, respectively. 
+     This cannot be specified if 'filter_string' is specified.
+     Example: {'state_name':{'oper':'isin','value':['CO']}, 
+     'WS':{'oper':'<','value':1}} 
+   * **filter_string:** This is a string used to filter the paired data sent 
+     to the statistics routine. The result is the same as using filter_dict.
+     This uses the pandas query method on the paired dataset.
+     This cannot be specified if 'filter_dict' is specified.
+     Example: state_name in ['CO'] and WS < 1
```

Comment:
Can we add something like the following?: "This option is only available for for surface and aircraft observations. For satellite observations, use the filter_dict option instead."